### ISSUE TITLE
paper-icon-button accessibility report 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-icon-button#4]
Here are the audit results: 

[Warning] Elements with onclick handlers must be focusable (9)
[Warning] Images should have an alt attribute (1)
[Warning] Text elements should have a reasonable contrast ratio (1)
Not applicable tests (8)
Passing tests (4)

The X icon does not have sufficient contrast. 

There are issues with keyboard navigation and focus. I am not able to access the icons just using tab, so I'm not able to press enter to activate. I can only activate via the mouse. 

The labels need to be more descriptive and clear for screen reader users. The alt text for the hearts, in particular, need to be improved.

On Mac with VoiceOver, I can access the icons fine and hear what the alt text descriptions are. However, on Chrome OS with ChromeVox, it's not possible to access the icons, only the written text.

**Comment by alice** [comment]
Suggestion: provide and document an `alt` attribute analogous to `input type=button` to explicitly allow user-provided labels.

**Comment by lpalmaro** [comment]
Friendly ping on progress

**Comment by notwaldorf** [comment]
Fixed the focus state, and now it looks like this:
![screen shot 2015-06-01 at 11 05 00 am](https://cloud.githubusercontent.com/assets/1369170/7919588/2f678010-084e-11e5-9760-011252c6e1fc.png)

The fact that the alt text is vague is a demo-specific thing (we set the alt text in the demo to menu/heart etc, so it's not really related to the element).

Can you please take another look and let me know if everything looks ok? Thanks!

**Comment by alice** [comment]
There's no `alt` attribute in these demos (or supported on these elements) - the demo uses a `title` attribute. `<input>` elements provide an `alt` attribute (http://www.w3.org/TR/2012/WD-html5-20121025/states-of-the-type-attribute.html#attr-input-alt) - my suggestion is that we provide and document a similar attribute for this element.
